Dockerfile.dev

```diff
@@ -39,9 +39,11 @@ RUN rm protoc-3.6.1-linux-x86_64.zip
 RUN mkdir -p /go
 ENV GOPATH /go
 ENV PATH $PATH:/usr/local/go/bin:/go/bin
-
 RUN go get github.com/golang/protobuf/protoc-gen-go && \
-go get -u golang.org/x/lint/golint
+go get golang.org/x/lint/golint && \
+go get golang.org/x/tools/cmd/cover && \
+go get github.com/mattn/goveralls
```

Comment:
As commented in `deploy.sh`, shall we delete all `go get`s in the Dockerfile?

Reply:
Sure, Done.